.github/workflows/ci.yml

```
@@ -48,9 +48,11 @@ jobs:
       - dart-tests-macos
       - dart-tests-macos-arm
       - dart-tests-linux
+      - dart-tests-linux20
       - flutter-desktop-tests-windows
       - flutter-desktop-tests-macos
       - flutter-desktop-tests-linux
+      - flutter-desktop-tests-linux20
```

```suggestion
      - flutter-desktop-tests-linux-ubuntu-20
```
can we rename all these to be more clear

---

Since there is no `linux20` thing